### DISCUSSION TITLE
How can I pass collection from bulk action to populate a create form?

### DISCUSSION THREAD

**Discussion by ismaeltj9** [filamentphp/filament#9374] [Help]
Hello,

I have this bulk action to get all selected row in a table and pass to a new create page:

Tables\Actions\BulkAction::make('Create Order')
    ->action(function (Collection $records) {
        redirect(OrderResource::getUrl('create'), [$records]);
    }),

How can I work with this collection in the create page? thank in advance.

**Comment by leandrocfe** [comment]
The CreatePage is a Livewire component. You could get the parameters via url
https://livewire.laravel.com/docs/url
